frontend/src/app/modules/availability/new-slot/event-detail/event-detail.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="container">
+    <p class="title text-inside">Link</p>
+    <div class="inputBlock">
+        <div class="inputStartBlock">
+            <p class="productName">productname.com/</p>
+        </div>
+        <input class="inputItem" [(ngModel)]="linkChoice" />
+    </div>
+
+    <p class="title text-inside">Welcome Message</p>
+
+    <textarea class="textAreaMessage" [(ngModel)]="welcomeMessage"></textarea>
+
+    <p class="title text-inside">Language</p>
+    <select name="select" class="selectBlock" [(ngModel)]="languageSelect">
+        <!--Supplement an id here instead of using 'name'-->
+        <option value="English" selected>English</option>
+        <option value="Ukrainian">Ukrainian</option>
+    </select>
+
+    <p class="title text-inside">Allowed bookings per Day</p>
+    <select name="select" class="selectBlock" [(ngModel)]="allowBookingSelect">
+        <!--Supplement an id here instead of using 'name'-->
+        <option value="one">1</option>
+        <option value="two" selected>2</option>
+        <option value="three">3</option>
+    </select>
+</div>
+
+<div class="checkbox-container">
+    <input type="checkbox" [(ngModel)]="isAllowBooker" [disabled]="!basicChoice" />
+    <span class="input-message text-inside"><p>Allow Booker to add additional guests</p></span>
+</div>
+
+<mat-radio-group aria-labelledby="example-radio-group-label" class="example-radio-group" [(ngModel)]="zoneChoice">
+    <mat-radio-button
+        disableRipple
+        class="example-radio-button"
+        *ngFor="let season of timeZoneChoices"
+        [value]="season"
+    >
```

Comment:
```suggestion
        [value]="season">
```

Your way is more as React style